onboard/catkin_ws/src/cv/scripts/test_images.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import rospy
+import cv2
+import os
+from custom_msgs.srv import ToggleModel
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge
+
+
+# Mock the camera by publishing the same image to a topic
+class DummyImagePublisher:
+
+    NODE_NAME = 'test_images'
+    CAMERA = 'left'
+    IMAGE_TOPIC = '/camera/{}'.format(CAMERA)
+    # IMAGE_TOPIC = '/test_images/image'
```

Comment:
An old comment that can be removed?

Reply:
Yes -- we just removed it. We will push those changes soon